addons/account/models/account_tax.py

```diff
@@ -1380,11 +1414,17 @@ def _prepare_base_line_tax_details(
             rounding_method = 'round_per_line'
         currency = base_line['currency'] or company.currency_id
         taxes_data = taxes._convert_to_dict_for_taxes_computation()
+        if (force_price_include := base_line['extra_context'].get('force_price_include')) is not None:
+            special_mode = 'total_included' if force_price_include else 'total_excluded'
```

Comment:
this is confusing: why using force_price_include instead of special_mode directly?

Reply:
@qdp-odoo It's just for the backward compatibility. I would like to  clean all those helpers for 17.4.